src/main/java/org/htwkvisu/controller/ApplicationController.java

```diff
@@ -78,4 +106,36 @@ public void initialize(URL location, ResourceBundle resources) {
     public void onResetViewClicked(MouseEvent ev) {
         canvas.centerView(new Point2D(51, 13)); // test value as an example!
     }
+
+    /**
+     * Refreshes Values for textFields
+     *
+     * @param ev Key Event
+     */
+    @FXML
+    public void onEnterPressed(KeyEvent ev) {
+        if(ev.getCode().equals(KeyCode.ENTER)){
+
+            NumericTextField temp = (NumericTextField) ev.getSource();
+
+            if(temp.getText().isEmpty()){
+                temp.setText(Integer.toString(temp.getDefaultValue()));
+            }
+
+            int value = Integer.parseInt(temp.getText());
+
+            if(pixelDensityTextField.equals(temp)){
+                canvas.setSamplingPixelDensity(value);
+                temp.setText( Integer.toString(canvas.getSamplingPixelDensity()));
```

Comment:
Please remove the `temp.setText` lines in the if else branches, you can easily call `temp.setText(Integer.toString(value));` outside of the if (line 137), without using the getter of each field.